package.json

```diff
@@ -127,6 +127,7 @@
     "mongodb": "4.0.0-beta.2",
     "mongodb-download-url": "^1.0.0",
     "mongodb-js-precommit": "^2.0.0",
+    "nock": "^13.0.11",
```

Comment:
Is this used somewhere?

Reply:
Yeah, here: https://github.com/mongodb-js/mongosh/blob/master/packages/build/src/barque.spec.ts#L3

Reply:
Hm … but it’s in the `build` package’s `devDependencies`, so we shouldn’t have to add it here … but then again, I remember that somebody else was also running into this, maybe Irina? Maybe we could remove the one in the build package then, so that we don’t have to install it twice?

Reply:
It was throwing an error without adding it to package.json, like missing dependencies. I will try to remove it from build then.